README.md

```diff
@@ -70,6 +70,49 @@ func CreateExamplePage(basePage coreModel.Page) model.ExamplePage {
 }
 ```
 
+### Referencing a local instance of dp-renderer in a docker container
+
+If you are running and developing within a docker container that includes references to the dp-renderer, for example, the [cantabular import journey](https://github.com/ONSdigital/dp-compose/tree/main/cantabular-import). Follow these steps to use a local instance:
+
+- Update the `go.mod` file in the relevant service to use the `replace statement` to point to your local dp-renderer instance
+e.g.
+
+```go
+replace "github.com/ONSdigital/dp-renderer" => "/Users/{yourName}/{yourDirectory}/github.com/ONSdigital/dp-renderer"
```

Comment:
We should remove `github.com/ONSdigital` from this replace path, unless it's required for the docker compose stuff to work. For instance, my local path to dp-renderer doesn't contain that substring: `/Users/ravipradhan/Documents/ons/dp-renderer`